metron-platform/metron-parsers/src/test/java/org/apache/metron/parsers/integration/SquidIntegrationTest.java

```diff
@@ -29,6 +29,11 @@ String getSensorType() {
   }
 
   @Override
```

Comment:
haha ok, you got it.  I'll adjust it tomorrow to be a different type of test and not piggyback on squid.  You're right, the parser tests are for testing the parsers, not testing the ancillary stuff, like kafka writers.  Thanks for keeping me honest, @ottobackwards